mibidata/mibi_image.py

```diff
@@ -342,6 +333,34 @@ def parse_aperture(value):
                 )
         return aperture
 
+
+    @staticmethod
+    def match_fov_folder(value, field, value_name,
+                         field_name):
+        """Enforces consistency between fov_id and folder.
+
+        Args:
+            value: Value to match with.
+            field: Field that needs to be updated to value.
+            value_name: String representing whether matching folder
+                        or fov_id field.
+            field_name: String representing whether updating folder
+                        or fov_id field.
+        Returns:
+            value: Value to set field to.
+        """
+        if field_name == 'fov_id' and value_name == 'folder':
+            value = value.split('/')[0]
+        if not field:
+            warnings.warn(
+                f'The "{field_name}" attribute is required if "{value_name}" '
```

Comment:
parameter not attribute

Reply:
This feels more like an exception than a warning. Do we ever want a user to be able to set folder? Should we just always derive it from fov_id? @jaytarolli what are you using in MIBIO for the -Filtered images?

Reply:
When an isobaric corrected or filtered image is saved, the default behavior of MIBI/O is to append a (user-customizable) string to the end of the `fov_id`, e.g.: fov-1-scan-1 - Filtered. `folder` is then set by replacing first part of the folder value with the `fov_id`.